src/features/expand-collapse-outdated-comments.js

```diff
@@ -0,0 +1,40 @@
+import select from 'select-dom';
+import {isMac, metaKey} from '../libs/utils';
+
+const keyName = isMac ? 'Alt' : 'Ctrl';
```

Comment:
On Mac it’s actually CMD, not alt

Reply:
I would actually expect to be Alt on both Windows and Mac. Ctrl/Cmd+Click imply open in a new tab. Alt + Click on MacOS is alternative behaviour which makes sense here. What do you guys think?